tools/cooja/apps/mspsim/build.gradle

```diff
@@ -0,0 +1,55 @@
+apply plugin: "java"
+
+sourceSets {
+    main {
+        java {
+            srcDir 'src'
+        }
+    }
+}
+
+task buildMspsimJar(type: Exec) {
+    environment System.getenv()
+    workingDir '../../../mspsim/'
+    commandLine 'sh', '-c', 'ant jar'
+}
```

Comment:
Not really, especially there is a lack of time, but I think that using 'sh' is a very bad idea. Especially if there are gradle provides the functionality natively: https://docs.gradle.org/current/userguide/ant.html